src/include/kdbnotification.h

```diff
@@ -34,7 +34,8 @@
  * (see
  * <a href="https://www.libelektra.org/plugins/internalnotification">its plugin docs</a>).
  *
- * - If no plugin is mounted at a desired position it is simply mounted.
+ * - If no plugin is mounted at a desired position the internalnotification
```

Comment:
Sentence still not clear.

Reply:
I rephrased it to "If there is no plugin mounted at a required position the internalnotification plugin is mounted at this position."